### ISSUE TITLE
[$15][MSFT-139] [Challenge Feed] No descriptive word for number of registrant while using the screen reader

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2835]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges
2. Switch on the screen reader
3. Press Tab key until the number of registrants is focused

**Expected Result**
It should be read-out by the screen reader > Number of Registrants <Value> or Number of Registrants

**Actual Result**
No descriptive word for a number of registrants while using the screen reader

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.3.1 Info and Relationships
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=6aa53cde%2D1eef%2D4bae%2Dbfe5%2Df5f5f79e9df4&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F49

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096047 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096047 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096047 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu PR https://github.com/topcoder-platform/community-app/pull/2945

**Comment by anonymousjaggu** [comment]
@PrakashDurlabhji please check feedback in PR

**Comment by PrakashDurlabhji** [comment]
@anonymousjaggu PR updated

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097138 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097138 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by anonymousjaggu** [comment]
@PrakashDurlabhji Please check the comment in `MR` and once the MR updated please remove the label `tcx_feedback` and add label `tcx_readyforreview`

**Comment by PrakashDurlabhji** [comment]
@anonymousjaggu PR updated

**Comment by drasticdpk** [comment]
Operating System: Windows
Browser: Chrome
Url : https://test-community-app.topcoder-dev.com/challenges/30055274/submit

**Comment by drasticdpk** [comment]
Operating System: Windows
Browser: Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097138<br/><br/>```This is an automated message for crazyk via Topcoder X```
